src/Confluent.Kafka/Impl/SafeKafkaHandle.cs

```diff
@@ -129,8 +129,8 @@ internal int Flush(int millisecondsTimeout)
             return OutQueueLength;
         }
 
-        internal long AddBrokers(string brokers)
```

Comment:
Any reason for this to have been defined as long? (int on librdkafka)
Switched back to int (I don't think we will ever add so many brokers :) )